marklogic-data-hub/src/test/java/com/marklogic/hub/flow/TemporalWriteTest.java

```diff
@@ -134,4 +172,21 @@ private void addFieldAndIndexes() {
         systemEndIndex.put("range-value-positions", false);
         new DatabaseManager(getHubConfig().getManageClient()).save(newNode.toString());
     }
+
+    private InputStream asInputStream(String value) {
+        return new ByteArrayInputStream(value.getBytes());
+    }
+
+    private DocumentMetadataHandle getDocumentMetadata() {
+        String[] uris = getDocUris();
+        assertTrue(uris.length > 0, "Expected at least one fruit URI, found zero");
+        return getHubClient().getStagingClient().newDocumentManager().readMetadata(uris[0], new DocumentMetadataHandle());
+    }
+
+    private String[] getDocUris() {
```

Comment:
Same thing, this is only used once - just move the one line of code here to getDocumentMetadata.

Reply:
Moved to getDocumentMetadata